features/nfc/stack/nfc_errors.h

```diff
@@ -0,0 +1,65 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * \file nfc_errors.h
+ * \copyright Copyright (c) ARM Ltd 2013
+ * \author Donatien Garnier
+ * \details NFC Error codes
+ */
+
+/** \addtogroup Core
+ *  @{
+ *  \name Error codes
+ *  @{
+ */
+
+#ifndef NFC_ERRORS_H_
+#define NFC_ERRORS_H_
+
+#define NFC_OK                    0   ///< No error
```

Comment:
cool, I should use these, let's make them into an enum so it's nice and type safe